runatlantis.io/docs/server-configuration.md

```diff
@@ -47,6 +47,18 @@ Values are chosen in this order:
 
 
 ## Flags
+### `--allow-commands`
+  ```bash
+  atlantis server --allow-commands=version,plan,apply,unlock,approve_policies
+  # or
+  ATLANTIS_ALLOW_COMMANDS='version,plan,apply,unlock,approve_policies'
+  ```
+  List of allowed commands to be run on the Atlantis server, Defaults to `version,plan,apply,unlock,approve_policies`
+
+  Notes:
+  * Accepts a comma separated list, ex. `command1,command2`.
+  * `version`, `plan`, `apply`, `unlock`, `approve_policies` and `import` are available.
```

Comment:
Isn't there also an `atlantis help` command? I don't know if it's an official command but i have some devs use it to dump out the help text. Will your pr still allow devs to comment that to get the same usage help text?

Reply:
help is not provided official command in comment_parser too. I guess it's ok to provide it by default implicitly.

Reply:
Perhaps it dumps out the usage content because it's a non existent command? Just wanted to make sure it will still work after this pr change

Reply:
ah, yes. help command always works fine 👌